code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -569,3 +579,12 @@
 	filling_color = "#A0522D"
 	tastes = list("chocolate" = 1)
 	foodtype = JUNKFOOD | SUGAR
+
+/obj/item/reagent_containers/food/snacks/riceball
+	name = "Onigiri"
+	desc = "A ball of rice with some light salt and a wrap of Kudzu skin."
```

Comment:
Typically onigiri have no salt but are instead filled with a rice-topping like umeboshi or furikake

Reply:
Wikia quote - Traditionally, an onigiri is filled with pickled ume (umeboshi), salted salmon, katsuobushi, kombu, tarako, or any other salty or sour ingredient as a natural preservative.